src/stores/UserStore.ts

```diff
@@ -204,48 +204,50 @@ export const useUserStore = defineStore('UserStore', () => {
 
   // Watch authentication changes from other tabs
   watch(isAuthenticated, () => {
-    console.log('isAuthenticated changed, current route:', router.currentRoute);
-    console.log('isAuthenticated changed, isAuthenticated:', isAuthenticated.value);
+    console.warn('UserStore::isAuthenticated() changed', {
+      currentRoute: router.currentRoute,
+      isAuthenticated: isAuthenticated.value
+    });
     if (isAuthenticated.value) {
       if (router.currentRoute.value.name === 'login') {
-        console.log('isAuthenticated became true while on login page');
+        console.log('UserStore::isAuthenticated() isAuthenticated became true while on login page');
 
         try {
           const maybeRedirect = queryRedirect.parse(router.currentRoute.value);
           const redirect = maybeRedirect.query.redirect;
           if (redirect.startsWith('/')) { // avoid possible third-party redirection
-            console.log('UserStore watch redirect:', redirect);
+            console.log('UserStore::isAuthenticated() redirect:', redirect);
             router.push(redirect);
             return;
           } else {
-            console.log('Refusing to redirect to', redirect);
+            console.log('UserStore::isAuthenticated() Refusing to redirect to', redirect);
           }
         } catch (e) {
           // do nothing
         }
-        console.log('UserStore watch push to index');
+        console.log('UserStore::isAuthenticated() watch push to index');
         router.push({
           name: 'index',
         });
         return;
       }
     } else {
-      console.log(router.currentRoute.value);
+      console.log('UserStore::isAuthenticated()', router.currentRoute.value);
       $reset();   // wipes tokens if tokens are expired
       if (router.currentRoute.value.name !== 'login') {
-        console.log('isAuthenticated became false while not on login page');
+        console.log('UserStore::isAuthenticated() isAuthenticated became false while not on login page');
 
         // Preserve destination
         const currentPath = router.currentRoute.value.fullPath;
-        console.log('current path:', currentPath);
+        console.log('UserStore::isAuthenticated() current path:', currentPath);
         if (currentPath !== '/') {
           const query: any = {};
           query.redirect = currentPath;
-          console.log('UserStore unauthenticated path not slash to login');
+          console.log('UserStore::isAuthenticated() UserStore unauthenticated path not slash to login');
           router.push({ name: 'login', query });
           return;
         }
-        console.log('UserStore unauthenticated to login');
+        console.log('UserStore::isAuthenticated() UserStore unauthenticated to login');
         router.push({
           name: 'login',
         });
```

Comment:
I kept these logs as `log` but I'm not sure if they can be changed to `debug`?

Reply:
Maybe we could remove these as well?